tests/e2e/coco-workload/create/chainsaw-test.yaml

```diff
@@ -0,0 +1,79 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2023 Authors of Nimbus
+
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: kyverno-adapter-policy-creation
+spec:
+  description: >
+   This test validates that creating a `coco-workload` SecurityIntent with SecurityIntentBinding generates the expected Kyverno Policy and its effect.
+  steps:
+    - name: "Create a sample deployment"
+      try:
+      - script:
+          content: kubectl create deployment -n $NAMESPACE nginx --image=nginx
+          check: 
+            (contains($stdout, 'deployment.apps/nginx created')): true
+
+    - name: "Verify deployment creation"
+      try:
+        - assert:
+            file: ../deploy-assert.yaml
+
+    - name: "Create a sample runtimeclass"
```

Comment:
How is confidential containers being installed ? Is kvm supported on the github runner node ?

Reply:
this is the sample runtime-class just to check the mutation of the deploy, as we discussed earlier. Which is the job of nimbus-kyverno right? @shivaccuknox